Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -238,78 +223,76 @@ async def fetch_event(
-                            async for line in res.content:
-                                stripped_line = line.strip()
-                                if stripped_line:
-                                    events_fetched += 1
-                                    try:
-                                        streaming_event = json.loads(stripped_line)
-                                        event_metadata = streaming_event.get('metadata', {})
-                                        event_creation_time = event_metadata.get('eventCreationTime', 0)
-                                        if not event_creation_time:
-                                            demisto.debug(
-                                                'Could not extract "eventCreationTime" field, using 0 instead. '
-                                                f'{streaming_event}')
-                                        else:
-                                            event_creation_time /= 1000
-                                        event_creation_time_dt = datetime.fromtimestamp(event_creation_time)
-                                        if event_creation_time_dt < first_fetch_time:
-                                            demisto.debug(f'Event with offset {event_metadata.get("offset")} '
-                                                          f'and creation time {event_creation_time} was skipped.')
-                                            continue
-                                        yield streaming_event
-                                    except json.decoder.JSONDecodeError:
-                                        demisto.debug(f'Failed decoding event (skipping it) - {str(stripped_line)}')
-                                else:
-                                    new_lines_fetched += 1
-                                if last_fetch_stats_print + timedelta(minutes=1) <= datetime.utcnow():
-                                    demisto.info(
-                                        f'Fetched {events_fetched} events and'
-                                        f' {new_lines_fetched} new lines'
-                                        f' from the stream in the last minute.')
-                                    events_fetched = 0
-                                    new_lines_fetched = 0
-                                    last_fetch_stats_print = datetime.utcnow()
-                    except Exception as e:
-                        demisto.debug(f'Failed to fetch event: {e} - Going to sleep for 10 seconds and then retry -'
-                                      f' {traceback.format_exc()}')
-                        await sleep(10)
+            demisto.debug(f'Starting event fetch loop. Container ID: {CONTAINER_ID}')
+            self.client = Client(
+                base_url=self.base_url, app_id=self.app_id, verify_ssl=self.verify_ssl, proxy=self.proxy
+            )
+            await self._discover_stream()
```

Comment:
This was previously inside a try/catch. Are we sure we want it outside of the try/catch?

Reply:
pushed the whole code in the loop into the try-except